webpack/assets/javascripts/react_app/components/ModelsTable/index.js

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import ModelsTable from './ModelsTable';
+import { schema } from './ModelsTableSchema';
+import reducer from './ModelsTableReducer';
+import * as actions from './ModelsTableActions';
+
+const mapStateToProps = ({ models }) => ({ ...models, schema });
```

Comment:
IMO I prefer to pass individual props instead passing the whole state object. It help me to stay in control and I can understand the code better.

Reply:
You mean, to take only the relevant attributes from the state that the component needs, right?
If so, I agree, it does make it more readable.
Thanks a lot!

Reply:
Yap :+1: